python/neuroglancer/pipeline/storage.py

```diff
@@ -222,15 +222,9 @@ def put_file(self, file_path, content, compress):
             with open(path, 'wb') as f:
                 f.write(content)
         except IOError:
-            try: 
-                # a raise condition is possible
-                # where the first try fails to create the file
-                # because the folder that contains it doesn't exists
-                # but when we try to create here, some other thread
-                # already created this folder
-                os.makedirs(os.path.dirname(path))
-            except OSError:
-                pass
+            dirname = os.path.dirname(path)
```

Comment:
Changed this to check the condition we care about. If there's some other kind of unknown IOError we probably want to know about it.